content/reference.bib

```diff
@@ -0,0 +1,7 @@
+@article{example2025sample,
```

Comment:
Add a small blog showing an example. We can also reference this MR on the blog so readers can quickly see what the required changes were.